### ISSUE TITLE
SQLiteException: no such table: tbl_discover_cards (code 1 SQLITE_ERROR)

### ISSUE THREAD

**Issue by loremattei** [wordpress-mobile/WordPress-Android#13016]
**Sentry Url**: https://sentry.io/share/issue/8901c9353bff433c8b1e56a1996420ab/
**User Count**: 79
**Count**: 92
**First Release**: org.wordpress.android@15.7-rc-2+918
**First Seen**: 2020-09-12T15:39:14Z
**Last Seen**: 2020-09-25T10:22:51Z
**24 Hours**: 54
**30 Days**: 92

```
SQLiteException: no such table: tbl_discover_cards (code 1 SQLITE_ERROR)
     at org.wordpress.android.datasets.ReaderDiscoverCardsTable.loadDiscoverCardsJsons(ReaderDiscoverCardsTable.kt:54)
     at org.wordpress.android.datasets.ReaderDiscoverCardsTableWrapper.loadDiscoverCardsJsons(ReaderDiscoverCardsTableWrapper.kt:8)
     at org.wordpress.android.ui.reader.repository.usecases.GetDiscoverCardsUseCase$get$2.invokeSuspend(GetDiscoverCardsUseCase.kt:32)
     at org.wordpress.android.ui.reader.repository.usecases.GetDiscoverCardsUseCase$get$2.invoke(null:10)
     at org.wordpress.android.ui.reader.repository.usecases.GetDiscoverCardsUseCase.get(GetDiscoverCardsUseCase.kt:31)
     at org.wordpress.android.ui.reader.repository.ReaderDiscoverDataProvider$reloadPosts$2.invokeSuspend(ReaderDiscoverDataProvider.kt:131)
     at org.wordpress.android.ui.reader.repository.ReaderDiscoverDataProvider$reloadPosts$2.invoke(null:10)
     at org.wordpress.android.ui.reader.repository.ReaderDiscoverDataProvider.reloadPosts(ReaderDiscoverDataProvider.kt:130)
     at org.wordpress.android.ui.reader.repository.ReaderDiscoverDataProvider$onUpdated$1.invokeSuspend(ReaderDiscoverDataProvider.kt:140)
...
(19 additional frame(s) were not displayed)
```

**Comment by loremattei** [comment]
This one doesn't seem to have a big impact right now, but since it seems to be new in the latest version, it may be worth it to investigate it more and fix it now.

**Comment by bummytime** [comment]
Fix #13073 was merged into 15.9. Closing.
